spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -57,9 +57,13 @@ call 04-channel `ChanOpenInit`. An error is returned if the controller portID is
 A `ChannelOpenInit` event is emitted which can be picked up by an offchain process such as a relayer.
 The account will be registered during the `OnChanOpenTry` step on the host chain.
 This function must be called after an `OPEN` connection is already established with the given connection identifier.
+The caller must provide the complete version. This MUST include the ICA version with complete metadata and it MAY include 
```

Comment:
Maybe you should define the concept of `ICA version` before using it here... It is actually explained further down in the [metadata negotiation section](https://github.com/cosmos/ibc/blob/8ed03c4124781fc74a00c6e6468a5cebe53c114a/spec/app/ics-027-interchain-accounts/README.md#metadata-negotiation)...

Reply:
Readability nit: nnd this is the first usage of the acronym `ICA`, maybe it would be good to explain what it means (either here or in the definitions section).